definitions.h

```diff
@@ -225,6 +230,14 @@ class Attributes
     public:
         //public member variabels
         PIXEL color;
+        double r, g, b, u, v;
+
+        vector<double> vAttr;
+        // This is what I would use to make the attributes
+        // more flexible, however, I ran out of time and
+        // wasn't able to fully implement it
+
```

Comment:
I'm guessing that you would store all your attributes in a vector. seems pretty interesting, I might try something like that later on.